Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -106,7 +107,9 @@ private void OnRelayMoveInput(EntityUid uid, BodyComponent component, ref MoveIn
         _humanoidSystem.SetLayersVisibility(bodyUid, layers, false, true, humanoid);
     }
 
-    public override HashSet<EntityUid> GibBody(EntityUid bodyId, bool gibOrgans = false, BodyComponent? body = null, bool deleteItems = false, bool deleteBrain = false)
+    public override HashSet<EntityUid> GibBody(EntityUid bodyId, bool gibOrgans = false, BodyComponent? body = null,
+        bool deleteItems = false, bool deleteBrain = false, GibbableComponent? gibbable = null, SoundSpecifier? gibSound = null,
+        bool launchGibs = true, Vector2? splatDirection = null, float splatModifier = 1, Angle splatCone = default)
```

Comment:
It should be one param per line as per https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#line-breaks-of-parameterargument-lists

Reply:
good catch, I forgot about that. I'm used to the way we did it at work which is keeping it together as much as possible.